pkg/ethutil/ethutil_test.go

```diff
@@ -62,32 +63,52 @@ func (s *EthUtilSuite) TestAddInput() {
 	sender := common.HexToAddress("0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266")
 	payload := common.Hex2Bytes("deadbeef")
 
-	inputIndex, err := AddInput(s.ctx, s.client, s.book, s.signer, payload)
-	if !s.Nil(err) {
-		s.logDevnetOutput()
-		s.T().FailNow()
+	indexChan := make(chan int)
+	errChan := make(chan error)
+
+	waitGroup := sync.WaitGroup{}
+	waitGroup.Add(1)
+
+	go func() {
+		waitGroup.Done()
+		inputIndex, err := AddInput(s.ctx, s.client, s.book, s.signer, payload)
+		if err != nil {
+			errChan <- err
+			return
+		}
```

Comment:
You should use `!s.Nil(err)` to properly use the suite

Reply:
Suite functions do not work inside Go routines. We need to "send them out" to threat them

Reply:
The change was removed from the PR, @GMKrieger